Weather Alert/Networking/ForecastRequest.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ForecastRequest.swift
+//  Weather Alert
+//
+//  Created by Jack Hopkins on 18/11/2016.
+//  Copyright © 2016 UKFast. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+class ForecastRequest: NetworkingRequest {
+    
+    init(cityId: Int) {
+        super.init()
+        path = "/forecast/daily?id=\(cityId)"
+    }
+    
+    func response(completionHandler: @escaping ([Forecast]) -> Void) {
+        
+        var objects: [Forecast] = []
+        
+        super.handleResponse() { result in
```

Comment:
If you're not overriding the function, calling `super` is useless as just calling `handleResponse` will still be called on the parent class.